src/main/java/com/google/sps/api/authorization/AuthorizationRequester.java

```diff
@@ -50,15 +50,13 @@
   public static final String CLIENT_SECRETS_PATH = "/client_secrets.json";
   public static final String ACCESS_TYPE = "offline";
   public static final String APPROVAL_PROMPT = "force";
-
-
```

Comment:
If possible, these reformatting changes for files without logic change should be submitted separately (e.g. before).

Reply:
Sorry. Is it ok if I look out to only reformat the code I was working with, and not the whole project?

Reply:
Yes